.travis.yml

```diff
@@ -24,6 +24,13 @@ jobs:
       script:
         - (cd proxy && cargo test --locked --no-default-features)
 
+    - language: rust
+      # Run proxy benchmarks. Requires nightly. Failures are non-fatal
+      rust: nightly
+      cache: cargo
+      script:
+        - (cd proxy && cargo bench --locked --no-default-features)
```

Comment:
Do we want to actual see the full benchmark results, or just test that they work? This will currently compile everything in release mode. An alternative is `cargo test --benches` I believe.